server/models/Version.ts

```diff
@@ -43,6 +45,8 @@ const VersionSchema: any = new Schema<Version>(
     built: { type: Boolean, default: false },
     managerApproved: { type: String, required: true, enum: approvalStateOptions, default: 'No Response' },
     reviewerApproved: { type: String, required: true, enum: approvalStateOptions, default: 'No Response' },
+    managerLastViewed: { type: Schema.Types.Mixed },
```

Comment:
I'm not super familiar with mongoose schema types, but is an ISO string considered a `Schema.Types.Date`?

Reply:
I believe because we're using the DateString it doesn't recognise it as a Date object and is failing to use the schema. I can double check but I believe that's why I changed it to this.